Packs/PrismaCloud/Playbooks/playbook-Prisma_Cloud_-_VM_Alert_Prioritization.yml

```diff
@@ -1490,6 +1490,19 @@ inputs:
   required: false
   description: The ID of the Slack channel to which to send the message.
   playbookInputQuery:
+inputSections:
+- inputs:
+  - JiraProjectName
+  - JiraIssueTypeName
+  - CloseIncidentAfterNotification
+  - SlackChannelName
+  - SlackChannelID
+  name: Incident Management
```

Comment:
In other playbooks (the PR omri opened - there was a section for ticket management)